### PR TITLE
Publish helm chart into gh_pages

### PR DESCRIPTION
**Pull request by k0da**
Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -18,3 +18,9 @@ jobs:
         repository: absaoss/samlet
         dockerfile: Dockerfile
         tag_with_ref: true
+    - uses: dave-mcconnell/helm-gh-pages-microservices@master
```
Comment (somaritane) [inline .github/workflows/release.yaml:21]: Maybe makes sense to also do some post-release smoke test, by deploying published chart from gh-pages to test kind cluster

**Review by somaritane — CHANGES_REQUESTED** [review]

**Review by somaritane — APPROVED** [review]

**Review by somaritane — APPROVED** [review]

**Review by ytsarev — COMMENTED** [review]
As an additional point to @somaritane comment, consider the implementation of the similar smoke test as in https://github.com/AbsaOSS/k8gb/blob/master/.github/workflows/release.yaml#L30-L39

File: .github/workflows/release.yaml (hunk 1 above)
Comment (k0da) [reply .github/workflows/release.yaml:21]: @somaritane done

**Review by somaritane — APPROVED** [review]
lgtm

**Pull request merged**